staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion/conversion_test.go

```diff
@@ -685,7 +685,7 @@ func expectConversionFailureMessage(id, message string) func(t *testing.T, ctc *
 		if err != nil {
 			t.Fatal(err)
 		}
-		for _, verb := range []string{"get", "list", "create", "udpate", "patch", "delete", "deletecollection"} {
+		for _, verb := range []string{"get", "list", "create", "update", "patch", "delete", "deletecollection"} {
 			t.Run(verb, func(t *testing.T) {
 				switch verb {
 				case "get":
```

Comment:
the update case looks wrong:
```
clients["v1beta2"].Update(context.TODO(), obj, metav1.UpdateOptions{})
```

that is passing a v1beta1 obj to a v1beta2 endpoint...

Reply:
@liggitt what would you suggest as a solution?  Change `obj, err := clients["v1beta1"].Create` above on line 684 to `v1beta2`?

Reply:
The thing about that though is that this function is about conversion, and I assume `convert` in this context means convert a `v1beta1` object to `v1beta2`?

Reply:
I would do something like the following:

```
// create a fixture object in the right version
v1beta2Obj := newConversionMultiVersionFixture(ns, id, "v1beta2")
// copy over the resourceVersion from the existing object so the update is allowed
v1beta2Obj.SetResourceVersion(obj.GetResourceVersion())
// try to update
clients["v1beta2"].Update(context.TODO(), v1beta2Obj, metav1.UpdateOptions{})
```

Reply:
@liggitt that worked, thank you, but now I'm getting a new error, which only occurs with the `update` method, in the second loop, when `subresource == "scale"`:
```
conversion_test.go:735: expected error with message returned 0 objects, expected 1, but got no kind is registered for the type v1.Scale in scheme "k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go:688"
```
Any ideas?

Reply:
you have to pass different objects depending on the subresource. the suggestion in https://github.com/kubernetes/kubernetes/pull/96967/files#r534504232 will work for "status".

for "scale", you'll have to pass a v1.Scale object

Reply:
@liggitt, `Update()` takes a `*unstructured.Unstructured`, so what would be the best way to create a `v1.Scale` and pass it to `Update()`?

Reply:
@liggitt

Reply:
@liggitt I'm lost here, can you help me to fix this (or just do it yourself if it's easier for you)?

Reply:
@sttts, can you help with this or route it to someone to shepherd fixup of the test?